.gitignore

```diff
@@ -15,8 +15,7 @@
 # vendor/
 
 # cosign stuff
-cosign.key
-cosign.pub
+cosign.sig
```

Comment:
Maybe we keep `cosign.key` and `cosign.pub` here for users, and move the keys below into `.github/workflows` if they're needed for the release?

Reply:
Ah, good call.